napari/layers/shapes/shapes.py

```diff
@@ -1387,6 +1387,21 @@ def to_labels(self, labels_shape=None):
 
         return labels
 
+    def _add_zarr_data(self, gp):
+        """Add layer data to zarr group."""
+        gp.array(
+            'data',
+            self.data,
+            shape=self.data.shape,
+            chunks=(128,) * len(self.data.shape),
```

Comment:
btw, the most common rule of thumb I've heard for chunking sizes is that they should be ~1MB. So it's probably worth writing a function that returns ~1MB chunks depending on the dimensions.